app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,35 @@
+class OrdersController < ApplicationController
+  load_and_authorize_resource
+
+  def index
+    @menus = Menu.all
+    @orders = params[:menu_id] ? Order.for_menu(params[:menu_id]) : Order.all
+  end
+
+  def new
+    @menu = Menu.find(params[:menu_id])
+  end
+
+  def create
+    menu_item_ids = MenuItem::TYPES_OF_DISHES.map {|dish_type| params["#{dish_type.to_s}_id"] }.compact
```

Comment:
this action is not forgivable though :)
move to service. As an application_service pattern I'd recommend https://github.com/pawurb/smart_init btw